aws/resources/cloudtrail.go

```diff
@@ -23,6 +23,7 @@ func (ct *CloudtrailTrail) getAll(c context.Context, configObj config.Config) ([
 		for _, trailInfo := range page.Trails {
 			if configObj.CloudtrailTrail.ShouldInclude(config.ResourceValue{
 				Name: trailInfo.Name,
+				Tags: util.ConvertTypesTagsToMap(trailInfo.Tags),
```

Comment:
The resource you are iterating over lacks the ['Tags' field](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/cloudtrail@v1.47.5/types#TrailInfo). You need to call [this method](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/cloudtrail#Client.ListTags) for each resource first.